frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -170,14 +169,10 @@ export default function Objective({
   };
 
   const onUploadFile = async (files, _objective, setError) => {
-    // we save draft one of two ways, depending on whether it is a
-    // recipient report or not
-    await onSaveDraft();
-
```

Comment:
we don't need to specifically save draft here anymore, we are still doing the lookup in the form data via the prop just to make sure we have the most up to date info.